docs/www/data-archiving.md

```diff
@@ -0,0 +1,95 @@
+---
+title: Data archiving
+order: 0
+---
+
+# Data archiving
+
+When producers send events to Redpanda, Redpanda stores the events on a local storage volume.
+Then, Redpanda replicates the data to other nodes,
+according to the requirements defined by the producer.
+These persistence and replication processes protect your data in the event of a node or cluster failure,
+but archiving data to cloud storage gives your data additional durability.
+
+After you configure data archiving, your data is uploaded to cloud storage.
+In the event of a data center failure, data corruption, or cluster migration,
+your data is safe in the cloud for you to recover to your cluster.
+
+> **_Notes:_**
+> - Archiving to Amazon AWS S3 is supported.
+>     Integration with other cloud storage providers is in development.
+> - Native data retrieval is in development.
```

Comment:
I don't know what this means. 

it should say something like "Tooling to rehydrate a cluster from S3 is in development, targeted for April 2021"

Reply:
If there is any reason this would slip, I would probably prefer: Restoring data from S3 is in development.

Reply:
Is restore from GCP already supported?

Reply:
They all use the same API. So restoring in general is not yet supported from any cloud, only archival is.